tests/functional/test_model_version_policy.py

```diff
@@ -0,0 +1,98 @@
+#
+# Copyright (c) 2019 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import sys
+import pytest
+from grpc.framework.interfaces.face.face import AbortionError
+sys.path.append(".")
+from conftest import get_model_metadata, model_metadata_response,\
+    wait_endpoint_setup  # noqa
+
+
+class TestModelVerPolicy():
+
+    @pytest.mark.parametrize("model_name, throw_error", [
```

Comment:
"all", "latest", "specific" as model_name?
You are using that in `get_model_metadata(...)` below and tests are passing. Am I missing something here?

Reply:
We launch OVMS with 3 models with different modelVersionPolicy. 
`all` means we expect that OVMS should serve all versions available under directory.
`specific` means we expect that OVMS should serve only specific versions.
`latest` means that we expect OVMS to support` x` the latest versions available in the catalog.
[Here](https://github.com/IntelAI/OpenVINO-model-server/blob/941535b8d05e4020dbb9ee11fd801f5fa947d02a/tests/functional/model_version_policy_config.json) you can check that model names matched to set modelVersionPolicy.

Reply:
Gotcha, I got a little confused with those names 😄